plugin/src/main/java/com/microsoft/java/bs/gradle/plugin/SourceSetsModelBuilder.java

```diff
@@ -137,13 +193,24 @@ public Object buildAll(String modelName, Project rootProject) {
             exclusionFromDependencies);
         collector.collectByConfigurationNames(getClasspathConfigurationNames(sourceSet));
         gradleSourceSet.setModuleDependencies(collector.getModuleDependencies());
-        gradleSourceSet.setProjectDependencies(collector.getProjectDependencies());
       });
     }
 
     return new DefaultGradleSourceSets(gradleSourceSets);
   }
 
+  private Set<Object> getArchiveSourcePaths(CopySpec copySpec) {
+    Set<Object> sourcePaths = new HashSet<>();
+    if (copySpec instanceof DefaultCopySpec) {
+      DefaultCopySpec defaultCopySpec = (DefaultCopySpec) copySpec;
+      sourcePaths.addAll(defaultCopySpec.getSourcePaths());
+      for (CopySpec child : defaultCopySpec.getChildren()) {
```

Comment:
I tested on an old gradle (4.9) and got:

```
Caused by: java.lang.NoSuchMethodError: org.gradle.api.internal.file.copy.DefaultCopySpec.getChildren()Ljava/util/Collection;
```

Reply:
I've pushed an update (minus a lot of the java test files as they're not needed, just the build setup is needed).
This should solve the 4.9 issue but I'm struggling to test this on early gradle versions - could you test it yourself again please?
Also rebased